fe/fe-core/pom.xml

```diff
@@ -1150,6 +1150,9 @@ under the License.
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <release>8</release>
+                </configuration>
```

Comment:
I suggest we define this in the parent POM, similar to how it’s done here: https://github.com/apache/doris/blob/master/fe/pom.xml#L130


```
<plugin>
    <groupId>org.apache.maven.plugins</groupId>
    <artifactId>maven-compiler-plugin</artifactId>
    <configuration>
        <release>8</release>
    </configuration>
</plugin>
```
This way, all the submodules will inherit this configuration. Otherwise, we can only ensure that the **fe-core** module works correctly.

Reply:
see desc of this PR

> not change all project to level 8 because java-common use sun.misc.unsafe cannot compile with level 8

Reply:
oops...